app/controllers/exams_controller.rb

```diff
@@ -17,9 +17,13 @@ def create
     end
   end
 
-  def index; end
+  def index
+    @exams = Exam.all.user_tests(current_user.id)
```

Comment:
```suggestion
    @exams = Exam.user_tests(current_user.id)
```
cơ mà chỗ này em có thể tận dụng association dc nè